main.tf

```
@@ -26,7 +36,9 @@ locals {
   # Determine connection mode and IP ranges
   connect_mode  = var.use_private_g_services ? "PRIVATE_SERVICE_ACCESS" : "DIRECT_PEERING"
   ip_cidr_range = var.use_private_g_services ? null : var.ip_cidr_range
-
+  # Read-replica for Redis memorystore
+  redis_replicas_mode = var.use_redis_replicas ? "READ_REPLICAS_ENABLED" : "READ_REPLICAS_DISABLED"
+  redis_replica_count = var.use_redis_replicas ? var.redis_replica_count : null
```

Everything in this module is related to `redis`. So the word `redis` is fairly redundant here.

```suggestion
  replica_mode  = var.use_redis_replicas ? "READ_REPLICAS_ENABLED" : "READ_REPLICAS_DISABLED"
  replica_count = var.use_redis_replicas ? var.redis_replica_count : null
```